backend/pkg/api/routes.go

```diff
@@ -86,11 +144,16 @@ func (api *API) routes() *chi.Mux {
 			r.Mount("/debug", chimiddleware.Profiler())
 		})
 
+		// Add our own gRPC-Gateway handlers
+		// TODO handle gatewayPathPrefix if this is needed
```

Comment:
i wonder what is with this. i saw it in enterprise repo, but don't understand the use case

Reply:
Not sure either @bojand should know

Reply:
i removed this comment for now.